### ISSUE TITLE
Companion Does not work with updated layout on 9anime

### ISSUE THREAD

**Issue by zrattz** [lap00zza/9anime-Companion#130]
- [ ] Bug
- [ ] Question
- [ ] Suggestion
- [x ] Other

New layout on 9anime so download wont work

**Comment by Alistair1231** [comment]
also mal integration not working properly...

**Comment by sliperhr** [comment]
yes,the companion didnt work with the new 9anime,hope you guys a new 9anime companion for the new 9anime,it would be great help

**Comment by amartyads** [comment]
yeah the addon doesnt detect anime and episode change anymore

**Comment by Alistair1231** [comment]
For the time being here is a very nice alternative for the mal integration (works on gogoanime, crunchyroll, 9anime, masterani.me and kissanime.to)
[KissAnimeList](https://github.com/lolamtisch/KissAnimeList) (it's a userscript, no addon... so you have to have smth like tampermonkey...) 
![image](https://user-images.githubusercontent.com/26165288/34584039-70a1c3b4-f19a-11e7-907b-7da4c890ea54.png)
